### PR TITLE
edits concepts

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

File: load-balancer/api-docs/getting-started/concepts.rst [hunk 1]
```diff
@@ -23,22 +23,22 @@ Virtual server
...
-for example ``172.16.200.210:80``. A virtual server's main function is to
-distribute traffic loads to a group of internal, back-end servers. All traffic
-is directed to the virtual server, and the server distributes the traffic
+such as ``172.16.200.210:80``. A virtual server's main function is to
+distribute traffic loads to a group of internal, back-end servers. The load balancer 
+directs all traffic to the virtual server, and the server distributes the traffic
 across the backend devices.
 
-Virtual servers are configured differently on |ADX| and |F5| devices.
+YOu configure virtual servers differently on |ADX| and |F5| devices.
```
Comment (humamahmad) [inline load-balancer/api-docs/getting-started/concepts.rst:31]: "You" is mentioned as "YOu".

**Comment by WilliamLoy** [comment]
build me

**Comment by WilliamLoy** [comment]
build me

**Comment by catlook** [comment]
re-editing the whole document.

**Pull request closed without merging**